PyTDF/Node.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import print_function
+import sys
+
+class Node(object):
+    """
+    A Class that represents
+    a graph Node
+
+    """
+    def __init__(self, _get_head, operation, tdf_object=None):
+        
+        if _get_head is None:
+            self._get_head = lambda : self
+        else:
+            self._get_head = _get_head
+        
+        self.operation = operation
+        self.next_nodes = []
+        self.cur_attr = ""
+        self.value = None
+        self.init_refcount = sys.getrefcount(self)
```

Comment:
So you save the initial count here so that you can use it as the lower value when pruning? Could we first try to understand why the initial value is 4? And why not always?